test/Api.Tests/TestStartup.cs

```diff
@@ -14,7 +12,9 @@ public void ConfigureTestServices(IServiceCollection services)
             base.ConfigureServices(services);
 
             // Register here the dependency injection services
-           // services.AddDbContext<ApplicationDbContext>(opts => opts.UseInMemoryDatabase());
```

Comment:
I basically understand what you told. I tried to replace the Dbtask in the similar way as mentioned in the example but it was not working and hence got suggestion from the admin stating that they do not consider this and it is better to have a check. The message he sent was,

Hi,

I am familiar with this issue. I can send you a fix tomorrow (with code example). Basically, you are calling .Migrate() in your Startup.cs. Because the testing library is using an in-memory database, it does not needs that call. What you can do is check whether the environment IsDevelopment or IsProduction before v calling Migrate(). The default environment on which the tests are executed is 'Test'. If you cannot fix it, I will send an example by tomorrow.

Kind regards,
Ivaylo
My Tested ASP.NET